src/lib/Sympa/DataSource/LDAP.pm

```diff
@@ -25,6 +25,9 @@ package Sympa::DataSource::LDAP;
 use strict;
 use warnings;
 
+use Net::LDAP::Control::Paged;
+use Net::LDAP::Constant qw( LDAP_CONTROL_PAGED );
```

Comment:
To avoid the namespace pollution, we'd be betterto refrain from using exported symbols. Use `Net::LDAP::Constant::LDAP_CONTROL_PAGED()`, although it is redundant.